Sources/StreamChatUI/Appearance+Formatters/MarkdownFormatter.swift

```diff
@@ -43,7 +44,80 @@ open class DefaultMarkdownFormatter: MarkdownFormatter {
 
     open func format(_ string: String) -> NSAttributedString {
         let markdownFormatter = SwiftyMarkdown(string: string)
-        markdownFormatter.code.fontName = Attributes.Code.fontName
+        modify(swiftyMarkdownFont: markdownFormatter.code, with: styles.codeFont)
+        modify(swiftyMarkdownFont: markdownFormatter.body, with: styles.bodyFont)
+        modify(swiftyMarkdownFont: markdownFormatter.link, with: styles.linkFont)
+        modify(swiftyMarkdownFont: markdownFormatter.h1, with: styles.h1Font)
+        modify(swiftyMarkdownFont: markdownFormatter.h2, with: styles.h2Font)
+        modify(swiftyMarkdownFont: markdownFormatter.h3, with: styles.h3Font)
+        modify(swiftyMarkdownFont: markdownFormatter.h4, with: styles.h4Font)
+        modify(swiftyMarkdownFont: markdownFormatter.h5, with: styles.h5Font)
+        modify(swiftyMarkdownFont: markdownFormatter.h6, with: styles.h6Font)
         return markdownFormatter.attributedString()
     }
+
+    private func modify(swiftyMarkdownFont: FontProperties, with font: MarkdownFont) {
+        if let fontName = font.name {
+            swiftyMarkdownFont.fontName = fontName
+        }
+        if let fontSize = font.size {
+            swiftyMarkdownFont.fontSize = fontSize
+        }
+        if let fontColor = font.color {
+            swiftyMarkdownFont.color = fontColor
+        }
+        if let fontStyle = font.styling?.asSwiftyMarkdownFontStyle() {
+            swiftyMarkdownFont.fontStyle = fontStyle
+        }
+    }
+}
+
+public struct MarkdownStyles {
+    public var bodyFont: MarkdownFont = .init()
+    public var codeFont: MarkdownFont = .init()
+    public var linkFont: MarkdownFont = .init()
+    public var h1Font: MarkdownFont = .init()
```

Comment:
Maybe would be nice to add a little documentation explain what are these exactly 🤔

Reply:
Added one 👍